src/Photos/Enums.cs

```diff
@@ -76,6 +76,9 @@ public enum PHVideoRequestOptionsDeliveryMode : long {
 	[Mac (10,12, onlyOn64 : true)]
 	[Native]
 	public enum PHCollectionListType : long {
+		[Deprecated (PlatformName.iOS, 13, 0)]
+		[Deprecated (PlatformName.TvOS, 13, 0)]
+		[Unavailable (PlatformName.MacOSX)]
```

Comment:
Generally, when do we use [Unavailable]? Is it specific to [Native] enums?

Reply:
`[Unavailable]` can be used anywhere but it's most useful in files processed by the generator (`bgen`/`btouch`). It basically tells the generator to skip the member for the specified platform.

One exception: enum values (like this case) are always generated on all platforms since 
* it's compiled as a numeric value, not a string symbol and won't be rejected; and
* it makes it easier to write code without defines, e.g. `#if __IOS__`

Still the IDE will read the attribute and show it as unavailable if someone is working on a Xamarin.Mac application.